betterdocs.gemspec

```diff
@@ -1,49 +1,49 @@
-# -*- encoding: utf-8 -*-
 # stub: betterdocs 0.7.1 ruby lib
 
 Gem::Specification.new do |s|
-  s.name = "betterdocs".freeze
```

Comment:
this file is auto-generated so it doesn't make sense to format it

Reply:
Probably did Cmd+Shift+S when it was open, I'll see if there is a .solargraphignore or to this effect